web/src/components/SecureAdminConsole.jsx

```diff
@@ -216,12 +225,12 @@ class SecureAdminConsole extends React.Component {
         token: token.replace(/"/g, ""),
         sessionRoles: Utilities.getCookie("session_roles"),
       };
-      const loggedIn = await this.completeLogin(loginData);
-      if (loggedIn) {
-        Utilities.removeCookie("token");
-        Utilities.removeCookie("session_roles");
-      }
-      return;
+      const loggedIn = this.completeLogin(loginData).then(() => {
```

Comment:
How is this working? it looks like it's trying to access a variable (`loggedIn`) in the outer scope.

Reply:
whoops